source/sonata_simulation.rst

```diff
@@ -96,6 +97,27 @@ example::
        "integration_method" : 2
   }
 
+Format of the electrodes_file
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The format of the weights file is described below:
+
+.. table::
+
+  ============================= ======================= ========== =============== ============= ==========================================
+  Group                         Dataset                 Type       Shape           Requirement   Description
+  ============================= ======================= ========== =============== ============= ==========================================
+  /electrodes/{electrodename}   layer                   utf8       1               Optional      Layer of the circuit in which {electrodename} is located
+  /electrodes/{electrodename}   position                float32    3               Optional      Position of {electrodename} in microns, in cartesian coordinates (3 x 1)
+  /electrodes/{electrodename}   region                  utf8       1               Optional      Region in which {electrodename} is located
+  /electrodes/{electrodename}   type                    utf8       1               Optional      Either EEG or LFP
```

Comment:
I think we should specify what `EEG` and `LFP` mean; ie how is the processing different

Reply:
I don't understand what the `{electrodename}` does; it appears to be metadata, but it's not clear where any of it is used. Maybe I'm missing something, but it doesn't seem to be referenced in the `/electrodes/{population_name}/*` or `/{population_name}/*` datasets

Reply:
`electrodename` is used in the code that generates the weights. Due to limitations in HDF5, the file is first created with values of 0 for the scaling factors. When these values are populated, `electrodename/positions` is used. The other, optional fields are just metadata